2024-10-12 03:16:58,510 - logging in using static token
2024-10-12 03:17:00,426 - Shard ID None has connected to Gateway (Session ID: 6b26087481642480f2276505a21caa08).
2024-10-12 03:18:42,262 - Ignoring exception in view <View timeout=None children=1> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 250, in close_callback
    del active_tickets[user.id]
        ~~~~~~~~~~~~~~^^^^^^^^^
KeyError: 1238326844013674540
2024-10-12 03:21:00,198 - logging in using static token
2024-10-12 03:21:02,130 - Shard ID None has connected to Gateway (Session ID: 104b2d893d0506950d12b48b6d4226f8).
2024-10-12 03:21:47,521 - User: md_abdullah25 | Query: sex | Response: Sorry, I couldn't find that Order ID. Please check and try again.
2024-10-12 03:22:09,906 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.danger: 4> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 279, in close_ticket_callback
    await private_channel.delete()
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\abc.py", line 867, in delete
    await self._state.http.delete_channel(self.id, reason=reason)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10003): Unknown Channel
2024-10-12 03:22:19,574 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.danger: 4> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 279, in close_ticket_callback
    await private_channel.delete()
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\abc.py", line 867, in delete
    await self._state.http.delete_channel(self.id, reason=reason)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10003): Unknown Channel
2024-10-12 03:22:23,173 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.danger: 4> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 279, in close_ticket_callback
    await private_channel.delete()
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\abc.py", line 867, in delete
    await self._state.http.delete_channel(self.id, reason=reason)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10003): Unknown Channel
2024-10-12 03:22:53,774 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.danger: 4> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 279, in close_ticket_callback
    await private_channel.delete()
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\abc.py", line 867, in delete
    await self._state.http.delete_channel(self.id, reason=reason)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10003): Unknown Channel
2024-10-12 03:23:14,091 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.danger: 4> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 279, in close_ticket_callback
    await private_channel.delete()
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\abc.py", line 867, in delete
    await self._state.http.delete_channel(self.id, reason=reason)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10003): Unknown Channel
2024-10-12 03:25:12,344 - logging in using static token
2024-10-12 03:25:14,441 - Shard ID None has connected to Gateway (Session ID: c364a2dd04c327407cf3d1fcfdb10a26).
2024-10-12 03:26:46,201 - User: md_abdullah25 | Query: 192.168.1.1 | Response: The status of IP `192.168.1.1` is \u2705 **ACTIVE**.
2024-10-12 03:29:03,081 - User: musabahmed9533 | Query: 20240925-20172118 | Response: **Order Details:**
**Name:** Md. Azraf Rosul
**Email:** azrafrosul33@gmail.com
**Phone Number:** +8801787870848
**IP Given:** Not found
**Order Date:** 25-09-2024
**Total Amount:** 160 BDT
**Package:** 1 Month - 100Mbps
**Validation Status:** Valid\U0001f44c
**Days Left:** 12
\U0001f195 New Updated IP Details \U0001f195:
IP: `49.0.41.6`
User: `REXFTP`
Port: `14852`
Password: `REXFTP86325`

2024-10-12 03:31:29,756 - logging in using static token
2024-10-12 03:31:31,762 - Shard ID None has connected to Gateway (Session ID: 0973905c4941338209f962589d91a567).
2024-10-12 03:31:51,055 - User: musabahmed9533 | Query: 20240925-20172118 | Response: **Order Details:**
**Name:** Md. Azraf Rosul
**Email:** azrafrosul33@gmail.com
**Phone Number:** +8801787870848
**IP Given:** Not found
**Order Date:** 25-09-2024
**Total Amount:** 160 BDT
**Package:** 1 Month - 100Mbps
**Validation Status:** Valid\U0001f44c
**Days Left:** 12
\U0001f195 New Updated IP Details \U0001f195:
IP: `49.0.41.6`
User: `REXFTP`
Port: `14852`
Password: `REXFTP86325`

2024-10-12 03:31:58,569 - User: md_abdullah25 | Query: 192.168.1.1 | Response: The status of IP `192.168.1.1` is \u274c **INACTIVE**.
2024-10-12 03:32:21,661 - User: md_abdullah25 | Query: 192.168.1.1 | Response: The status of IP `192.168.1.1` is \u2705 **ACTIVE**.
2024-10-12 03:33:09,215 - User: md_abdullah25 | Query: 20240925-20172118 | Response: **Order Details:**
**Name:** Md. Azraf Rosul
**Email:** azrafrosul33@gmail.com
**Phone Number:** +8801787870848
**IP Given:** Not found
**Order Date:** 25-09-2024
**Total Amount:** 160 BDT
**Package:** 1 Month - 100Mbps
**Validation Status:** Valid\U0001f44c
**Days Left:** 12
\U0001f195 New Updated IP Details \U0001f195:
IP: `49.0.41.6`
User: `REXFTP`
Port: `14852`
Password: `REXFTP86325`

2024-10-12 03:36:37,592 - User: md_abdullah25 | Query: 20240925-20172118 | Response: **Order Details:**
**Name:** Md. Azraf SEX
**Email:** azrafrosul33@gmail.com
**Phone Number:** +8801787870848
**IP Given:** Not found
**Order Date:** 25-09-2024
**Total Amount:** 160 BDT
**Package:** 1 Month - 100Mbps
**Validation Status:** Valid\U0001f44c
**Days Left:** 12
\U0001f195 New Updated IP Details \U0001f195:
IP: `49.0.41.6`
User: `REXFTP`
Port: `14852`
Password: `REXFTP86325`

2024-10-12 03:40:11,480 - logging in using static token
2024-10-12 03:40:13,653 - Shard ID None has connected to Gateway (Session ID: 3a723692aedc6aa8c3cf665001305ec1).
2024-10-12 03:43:02,739 - logging in using static token
2024-10-12 03:43:04,803 - Shard ID None has connected to Gateway (Session ID: 01bdc30f839f4b527296cec453634d05).
2024-10-12 03:43:41,456 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 159, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 233, in request
    raise DiscordServerError(response, data)
discord.errors.DiscordServerError: 503 Service Unavailable (error code: 0): upstream connect error or disconnect/reset before headers. reset reason: overflow
2024-10-12 03:43:54,476 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 140, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:47:38,797 - logging in using static token
2024-10-12 03:47:40,814 - Shard ID None has connected to Gateway (Session ID: b5b271840555fa51f43f4eabf2dc60e7).
2024-10-12 03:48:11,020 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 147, in yes_callback
    await interaction.response.defer(ephemeral=True)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 709, in defer
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:48:44,836 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 147, in yes_callback
    await interaction.response.defer(ephemeral=True)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 709, in defer
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:50:20,010 - logging in using static token
2024-10-12 03:50:22,224 - Shard ID None has connected to Gateway (Session ID: d023bfbda03f6b5724d01b5f8d21887e).
2024-10-12 03:51:15,867 - logging in using static token
2024-10-12 03:51:17,751 - Shard ID None has connected to Gateway (Session ID: 46d4786f1c119d944fadb3b0fc61ed76).
2024-10-12 03:52:16,581 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='IP Status' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 291, in ip_status_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 233, in request
    raise DiscordServerError(response, data)
discord.errors.DiscordServerError: 503 Service Unavailable (error code: 0): upstream connect error or disconnect/reset before headers. reset reason: connection timeout
2024-10-12 03:52:17,764 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.danger: 4> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 297, in close_ticket_callback
    await interaction.response.send_message("Thank you! The ticket is now closed.")
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 233, in request
    raise DiscordServerError(response, data)
discord.errors.DiscordServerError: 503 Service Unavailable (error code: 0): upstream connect error or disconnect/reset before headers. reset reason: overflow
2024-10-12 03:52:27,906 - logging in using static token
2024-10-12 03:52:29,845 - Shard ID None has connected to Gateway (Session ID: fc60d7458179c5155d1025e4aaa23d4d).
2024-10-12 03:53:11,161 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 153, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:53:13,085 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.primary: 1> url=None disabled=False label='My proxy is not working' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 238, in proxy_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:53:20,028 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.danger: 4> url=None disabled=False label='No' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 329, in no_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:53:37,042 - logging in using static token
2024-10-12 03:53:38,896 - Shard ID None has connected to Gateway (Session ID: 67f9fcda232cd4a3ddf2f8d56d8833d2).
2024-10-12 03:54:08,504 - logging in using static token
2024-10-12 03:54:10,405 - Shard ID None has connected to Gateway (Session ID: 99c7757fdf8193d7a789a3157360716a).
2024-10-12 03:54:19,898 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 154, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:54:47,344 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 154, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 233, in request
    raise DiscordServerError(response, data)
discord.errors.DiscordServerError: 503 Service Unavailable (error code: 0): upstream connect error or disconnect/reset before headers. reset reason: overflow
2024-10-12 03:56:22,033 - logging in using static token
2024-10-12 03:56:24,121 - Shard ID None has connected to Gateway (Session ID: 74aaa442fc217b45d78a8eaa4f8b7e27).
2024-10-12 03:56:46,039 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 153, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:57:03,797 - logging in using static token
2024-10-12 03:57:05,691 - Shard ID None has connected to Gateway (Session ID: c4288dfbfb59fc4be1e02af15465e770).
2024-10-12 03:57:26,553 - logging in using static token
2024-10-12 03:57:28,419 - Shard ID None has connected to Gateway (Session ID: 0e07459604ae23753a38d1b29729ab2b).
2024-10-12 03:58:52,262 - logging in using static token
2024-10-12 03:58:59,320 - Shard ID None has connected to Gateway (Session ID: bf3c7b72f35808db3249c2dd4f5b4a5d).
2024-10-12 03:59:18,372 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 153, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 03:59:37,067 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 153, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 233, in request
    raise DiscordServerError(response, data)
discord.errors.DiscordServerError: 503 Service Unavailable (error code: 0): upstream connect error or disconnect/reset before headers. reset reason: overflow
2024-10-12 04:00:47,494 - logging in using static token
2024-10-12 04:00:49,580 - Shard ID None has connected to Gateway (Session ID: 3b9961de13761e720183c62ed32843d1).
2024-10-12 04:01:03,197 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 153, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 04:01:19,456 - logging in using static token
2024-10-12 04:01:21,523 - Shard ID None has connected to Gateway (Session ID: 4b528e9d937d4d575e30ace850aa90fe).
2024-10-12 04:01:55,983 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 153, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 04:02:07,418 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\tempCodeRunnerFile.py", line 153, in yes_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 233, in request
    raise DiscordServerError(response, data)
discord.errors.DiscordServerError: 503 Service Unavailable (error code: 0): upstream connect error or disconnect/reset before headers. reset reason: overflow
2024-10-12 04:06:22,450 - logging in using static token
2024-10-12 04:06:24,450 - Shard ID None has connected to Gateway (Session ID: e911b5babc1c4c2602c402915a590eb7).
2024-10-12 04:06:37,891 - Ignoring exception in view <View timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Yes' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 147, in yes_callback
    await interaction.response.defer(ephemeral=True)
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 709, in defer
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 04:06:54,979 - Ignoring exception in view <View timeout=None children=3> for item <Button style=<ButtonStyle.primary: 1> url=None disabled=False label='My proxy is not working' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 248, in proxy_callback
    await interaction.response.send_message(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 04:06:59,235 - User: md_abdullah25 | Query: 20240925-20172118 | Response: IP address `20240925-20172118` not found in our records.
2024-10-12 04:07:13,970 - Ignoring exception in view <View timeout=None children=1> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Close Ticket' emoji=None row=None sku_id=None>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\ui\view.py", line 430, in _scheduled_task
    await item.callback(interaction)
  File "i:\Torrents\Musab\DiscordBT\main.py", line 609, in close_callback
    await interaction.response.send_message("Thank you! The ticket is now closed.")
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\interactions.py", line 855, in send_message
    await adapter.create_interaction_response(
  File "I:\Torrents\Musab\DiscordBT\.venv\Lib\site-packages\discord\webhook\async_.py", line 221, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction
2024-10-12 04:09:13,333 - logging in using static token
2024-10-12 04:09:15,326 - Shard ID None has connected to Gateway (Session ID: 0d25bec70d4e148ac198ff60f53cd31a).
2024-10-12 04:09:39,021 - User: md_abdullah25 | Query: dhon | Response: IP address `dhon` not found in our records.
2024-10-12 15:18:52,576 - logging in using static token
2024-10-12 15:18:54,562 - Shard ID None has connected to Gateway (Session ID: 2f090636b2b0ac22cf0e491ac101602c).
2024-10-12 15:19:52,261 - logging in using static token
2024-10-12 15:19:54,251 - Shard ID None has connected to Gateway (Session ID: 5fdcac085c5be5fd910b4cff8b7840a0).
2024-10-12 15:20:38,395 - logging in using static token
2024-10-12 15:20:40,387 - Shard ID None has connected to Gateway (Session ID: 48c1348502259cc4c0964220eeb7aa2b).
2024-10-12 15:21:37,094 - logging in using static token
2024-10-12 15:21:39,131 - Shard ID None has connected to Gateway (Session ID: 7bf9c2ff74fde4ae8641930d52275626).
2024-10-12 15:22:29,321 - logging in using static token
2024-10-12 15:22:31,116 - Shard ID None has connected to Gateway (Session ID: 9f610f71243a5c059de71838b9312e93).
2024-10-12 15:24:05,909 - logging in using static token
2024-10-12 15:24:07,853 - Shard ID None has connected to Gateway (Session ID: 0d3de797271e519603f2acbfe49f7d67).
2024-10-12 15:24:27,258 - User: musabahmed9533 | Query: 20240826-17062753 | Response: **Order Details:**
**Name:** Musab
**Email:** ismatjoha54@gmail.com
**Phone Number:** +8801913990196
**IP Given:** Not found
**Order Date:** 26-08-2024
**Total Amount:** 80 BDT
**Package:** 1 Month - 50Mbps
**Validation Status:** Expired\u26d4
**Days Left:** -18
\u26a0\ufe0f You need to renew to get updated Proxy, as your validation has 5 days or fewer left or expired. Please renew.

2024-10-12 15:30:28,240 - logging in using static token
2024-10-12 15:30:30,573 - Shard ID None has connected to Gateway (Session ID: 6eacb033c784738fb28e600160d9e079).
2024-10-12 15:30:52,484 - Ignoring exception in on_message
Traceback (most recent call last):
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "c:\DiscordBT\main.py", line 417, in on_message
    order_date = datetime.strptime(order_date_str, '%d-%m-%Y')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 554, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024/10/11' does not match format '%d-%m-%Y'
2024-10-12 15:31:37,302 - Ignoring exception in on_message
Traceback (most recent call last):
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "c:\DiscordBT\main.py", line 417, in on_message
    order_date = datetime.strptime(order_date_str, '%d-%m-%Y')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 554, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024/08/07' does not match format '%d-%m-%Y'
2024-10-12 15:34:42,450 - Ignoring exception in on_message
Traceback (most recent call last):
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "c:\DiscordBT\main.py", line 417, in on_message
    order_date = datetime.strptime(order_date_str, '%d-%m-%Y')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 554, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024/08/22' does not match format '%d-%m-%Y'
2024-10-12 15:35:01,161 - logging in using static token
2024-10-12 15:35:02,968 - Shard ID None has connected to Gateway (Session ID: 8492bd1b3cd58bb7a28ac3377f46faec).
2024-10-12 15:35:19,048 - Ignoring exception in on_message
Traceback (most recent call last):
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "c:\DiscordBT\main.py", line 417, in on_message
    order_date = datetime.strptime(order_date_str, '%d-%m-%Y')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 554, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024/08/22' does not match format '%d-%m-%Y'
2024-10-12 15:36:28,389 - logging in using static token
2024-10-12 15:36:30,421 - Shard ID None has connected to Gateway (Session ID: 24b05731305f4a0cd87e11458e1382fa).
2024-10-12 15:36:49,435 - Ignoring exception in on_message
Traceback (most recent call last):
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "c:\DiscordBT\main.py", line 417, in on_message
    order_date = datetime.strptime(order_date_str, '%d-%m-%Y')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 554, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024/08/22' does not match format '%d-%m-%Y'
2024-10-12 15:37:27,757 - logging in using static token
2024-10-12 15:37:29,630 - Shard ID None has connected to Gateway (Session ID: 6631ea6feb6e800f15636f7096dc0626).
2024-10-12 15:37:52,957 - User: musabahmed9533 | Query: 20240826-17062753 | Response: **Order Details:**
**Name:** Musab
**Email:** ismatjoha54@gmail.com
**Phone Number:** +8801913990196
**IP Given:** Not found
**Order Date:** 26-08-2024
**Total Amount:** 80 BDT
**Package:** 1 Month - 50Mbps
**Validation Status:** Expired\u26d4
**Days Left:** -18
\u26a0\ufe0f You need to renew to get updated Proxy, as your validation has 5 days or fewer left or expired. Please renew.

2024-10-12 18:10:55,180 - logging in using static token
2024-10-12 18:10:57,162 - Shard ID None has connected to Gateway (Session ID: dba3bad8d5b33b386f261baedc2fdf7c).
2024-10-12 18:11:41,513 - User: musabahmed9533 | Query: 20240928-22131572 | Response: **Order Details:**
**Name:** Md Sabbir
**Email:** smd146899@gmail.com
**Phone Number:** +8801814739612
**IP Given:** 103.35.109.22
**Order Date:** 28-09-2024
**Total Amount:** 70 BDT
**Package:** 1 Month - 40Mbps
**Validation Status:** Valid\U0001f44c
**Days Left:** 15
\U0001f195 New Updated IP Details \U0001f195:
IP: `103.35.109.22`
User: `REXFTP`
Port: `4040`
Password: `REXFTP563258`

2024-10-12 18:19:51,830 - User: musabahmed9533 | Query: 20240826-17062757 | Response: Sorry, I couldn't find that Order ID. Please check and try again.
2024-10-12 18:21:55,094 - User: musabahmed9533 | Query: 20240826-17062757 | Response: **Order Details:**
**Name:** Abul
**Email:** ismatjoha54@gmail.com
**Phone Number:** +8801913990196
**IP Given:** Not found
**Order Date:** 26-08-2024
**Total Amount:** 80 BDT
**Package:** 1 Month - 50Mbps
**Validation Status:** Expired\u26d4
**Days Left:** -18
\u26a0\ufe0f You need to renew to get updated Proxy, as your validation has 5 days or fewer left or expired. Please renew.

2024-10-12 18:31:03,166 - Ignoring exception in on_message
Traceback (most recent call last):
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\site-packages\discord\client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "c:\DiscordBT\main.py", line 417, in on_message
    order_date = datetime.strptime(order_date_str, '%d-%m-%Y')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 554, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\hp\AppData\Local\Programs\Python\Python312\Lib\_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024-10-11' does not match format '%d-%m-%Y'
2024-10-12 18:50:37,939 - logging in using static token
2024-10-12 18:50:39,821 - Shard ID None has connected to Gateway (Session ID: 15ec5e9f707e0f8716a0337efe4f6c4d).
2024-10-12 18:50:57,403 - User: musabahmed9533 | Query: 20240811-12105784 | Response: **Order Details:**
**Name:** Tanvir hasan
**Email:** N/A
**Phone Number:** N/A
**IP Given:** Not found
**Order Date:** 11-08-2024
**Total Amount:** 70 BDT
**Package:** 1 Month - 40Mbps
**Validation Status:** Expired\u26d4
**Days Left:** -33
\u26a0\ufe0f You need to renew to get updated Proxy, as your validation has 5 days or fewer left or expired. Please renew.

2024-10-12 20:51:56,111 - logging in using static token
2024-10-12 20:51:58,106 - Shard ID None has connected to Gateway (Session ID: aedab6d59dcb9d8ebe5e9f94ef6f4ce6).
2024-10-12 20:53:14,841 - User: musabahmed9533 | Query: 202.4.123.74 | Response: IP address `202.4.123.74` not found in our records.
2024-10-12 20:53:30,702 - User: musabahmed9533 | Query: 49.0.41.6 | Response: IP address `49.0.41.6` not found in our records.
2024-10-16 02:02:51,310 - logging in using static token
2024-10-16 02:02:53,479 - Shard ID None has connected to Gateway (Session ID: 9391bb790959dc08cff08b1a89690291).
2024-10-16 02:04:48,471 - logging in using static token
2024-10-16 02:04:50,370 - Shard ID None has connected to Gateway (Session ID: 30460b0636f048e443039ee904553b1c).
2024-10-16 02:46:38,023 - logging in using static token
2024-10-16 02:46:40,006 - Shard ID None has connected to Gateway (Session ID: e76427fa1b0ffeca5320ddf5d57236c0).
2024-10-16 02:47:53,417 - logging in using static token
2024-10-16 02:47:55,398 - Shard ID None has connected to Gateway (Session ID: 5e4781bd18ab35b8b918a8d9210454f0).
2024-10-16 02:48:39,290 - logging in using static token
2024-10-16 02:48:41,166 - Shard ID None has connected to Gateway (Session ID: 708bf401ea7fad1585ef9d91f6df326f).
2024-10-16 02:48:56,430 - logging in using static token
2024-10-16 02:48:58,330 - Shard ID None has connected to Gateway (Session ID: afb1554ed3dafb271d94e72b7b414aef).
2024-10-16 02:50:22,082 - logging in using static token
2024-10-16 02:50:24,108 - Shard ID None has connected to Gateway (Session ID: 96a2cd6f7911edb1c711090dcb9522aa).
2024-10-16 02:51:36,037 - logging in using static token
2024-10-16 02:51:38,081 - Shard ID None has connected to Gateway (Session ID: 5524cd4e3885a4da88a85aa536f77661).
2024-10-16 02:53:40,075 - logging in using static token
2024-10-16 02:53:42,077 - Shard ID None has connected to Gateway (Session ID: 1ffdeb32325ab3fa718e96e59369a7fa).
2024-10-16 02:54:13,631 - logging in using static token
2024-10-16 02:54:15,461 - Shard ID None has connected to Gateway (Session ID: 6a2bb21e2d4cafd8c3e3358c09065179).
2024-10-16 02:56:47,845 - logging in using static token
2024-10-16 02:56:49,754 - Shard ID None has connected to Gateway (Session ID: 3a961539afdbe5e2a4db63ea9972dbfc).
2024-10-16 02:57:31,603 - logging in using static token
2024-10-16 02:57:33,440 - Shard ID None has connected to Gateway (Session ID: ab3728d03724ca7300a67524dbeb15c0).
2024-10-16 02:58:40,306 - logging in using static token
2024-10-16 02:58:42,147 - Shard ID None has connected to Gateway (Session ID: c9b4f40cbfb148238fd37d458d01d20f).
2024-10-16 02:59:35,035 - logging in using static token
2024-10-16 02:59:37,050 - Shard ID None has connected to Gateway (Session ID: d8fee73305ea1bb7e1b387d8477f3a2b).
2024-10-16 03:00:57,849 - logging in using static token
2024-10-16 03:00:59,906 - Shard ID None has connected to Gateway (Session ID: 8fbf20a3fb17edf21fcc463e92b82df7).
2024-10-16 03:02:25,000 - logging in using static token
2024-10-16 03:02:26,921 - Shard ID None has connected to Gateway (Session ID: de84d14e274afca33d28b28e4de2cbde).
2024-10-16 03:05:33,697 - logging in using static token
2024-10-16 03:05:35,621 - Shard ID None has connected to Gateway (Session ID: 76c112be915f54af49a92f239042f606).
2024-10-16 03:06:05,414 - User: md_abdullah25 | Query: 103.171.143.137 | Response: The status of IP `103.171.143.137` is \u274c **INACTIVE**.
2024-10-16 03:06:41,864 - User: md_abdullah25 | Query: 103.171.143.137 | Response: The status of IP `103.171.143.137` is \u274c **INACTIVE**.
2024-10-16 03:07:19,894 - User: md_abdullah25 | Query: 103.171.143.137 | Response: The status of IP `103.171.143.137` is \u2705 **ACTIVE**.
2024-10-16 03:19:05,072 - logging in using static token
2024-10-16 03:19:07,153 - Shard ID None has connected to Gateway (Session ID: 9a81bd315a0ef17d3da9319e7314bf35).
2024-10-16 03:19:49,608 - logging in using static token
2024-10-16 03:19:51,445 - Shard ID None has connected to Gateway (Session ID: e2ce060ccde970f56b3d96dc62dd010e).
2024-10-16 03:20:07,485 - User: md_abdullah25 | Query: 103.171.143.137 | Response: The status of IP `103.171.143.137` is \u274c **INACTIVE**.
2024-10-16 03:21:14,088 - Task exception was never retrieved
future: <Task finished name='Task-12' coro=<watch_ip_status() done, defined at I:\Torrents\Musab\DiscordBT\main.py:370> exception=JSONDecodeError('Expecting value: line 1 column 1 (char 0)')>
Traceback (most recent call last):
  File "I:\Torrents\Musab\DiscordBT\main.py", line 377, in watch_ip_status
    ip_status_data = load_ip_status()
                     ^^^^^^^^^^^^^^^^
  File "I:\Torrents\Musab\DiscordBT\main.py", line 50, in load_ip_status
    return json.load(file)
           ^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Program Files\Python312\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-10-16 03:21:35,889 - logging in using static token
2024-10-16 03:21:37,784 - Shard ID None has connected to Gateway (Session ID: 25f1a591e9f4db8bf9b01f6c36767112).
2024-10-16 03:22:08,173 - logging in using static token
2024-10-16 03:22:10,075 - Shard ID None has connected to Gateway (Session ID: 60699fcab17180e6f8511ba9e9389e6e).
2024-10-16 03:23:29,002 - User: md_abdullah25 | Query: 103.171.143.137 | Response: The status of IP `103.171.143.137` is \u274c **INACTIVE**.
2024-10-16 03:25:19,980 - User: md_abdullah25 | Query: 20240809-01103851 | Response: **Order Details:**
**Name:** Abdullah Alamin
**Email:** N/A
**Phone Number:** N/A
**IP Given:** Not found
**Order Date:** 09-08-2024
**Total Amount:** 80 BDT
**Package:** 1 Month - 50Mbps
**Validation Status:** Expired\u26d4
**Days Left:** -39
\u26a0\ufe0f You need to renew to get updated Proxy, as your validation has 5 days or fewer left or expired. Please renew.

2024-10-16 03:30:07,060 - logging in using static token
2024-10-16 03:30:09,162 - Shard ID None has connected to Gateway (Session ID: a88010f337c06698211efae86d2d87ae).
2024-10-16 04:34:33,174 - logging in using static token
2024-10-16 04:34:35,530 - Shard ID None has connected to Gateway (Session ID: 78091e2df0efd9ff4c14bdd740f1e173).
2024-10-17 16:44:21,116 - logging in using static token
2024-10-17 16:44:23,419 - Shard ID None has connected to Gateway (Session ID: db938fc32e192e5a86d1ff79695011ec).
2024-10-18 12:27:57,811 - logging in using static token
2024-10-18 12:28:01,613 - Shard ID None has connected to Gateway (Session ID: ad5874eab00a95886881ce4c3c2701e6).
2024-10-18 12:28:22,734 - User: musabahmed9533 | Query: 20241015-22334229 | Response: **Order Details:**
**Name:** Mamunul Haque Arnab
**Email:** mh.arnab015@gmail.com
**Phone Number:** +8801775780631
**IP Given:** 49.0.41.6
**Order Date:** 15-10-2024
**Total Amount:** 80 BDT
**Package:** 1 Month - 50Mbps
**Validation Status:** Valid\U0001f44c
**Days Left:** 26
\U0001f195 New Updated IP Details \U0001f195:
IP: `49.0.41.6`
User: `REXFTP`
Port: `14852`
Password: `REXFTP86325`

2024-10-18 16:40:01,669 - logging in using static token
2024-10-18 17:24:05,130 - logging in using static token
2024-10-18 17:24:08,576 - Shard ID None has connected to Gateway (Session ID: 8620b044db8063c7a1d11eb98bbb8059).
2024-10-18 17:36:40,648 - logging in using static token
2024-10-18 17:36:45,082 - Shard ID None has connected to Gateway (Session ID: 57df5603180d4bc7c236c38f168b1383).
